source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,105 @@
+
+Here are some examples of accessiblity straight from Video.js:
+* The `MuteToggle` `<button>`:
+  * Has `aria-live` set to `polite`, which means that the screen reader should wait until the user is idle to send `aria-valuenow` updates to the user
+  * Has control text of "Mute" or "Unmute" which indicates the current status of the button to the user
+* The `VolumeBar` slider `<div>`:
+  * Has a `role` attribute with a value of `slider` like this: `role="slider"`
+  * Has a `tabIndex` attribute
+  * Has `EventHandlers` that listen for:
+    * The up and right arrow keys to increase the volume and the slider percentage
+    * The down and left arrow keys to decrease the volume and the slider percentage
+  * Has `aria-label` of "volume level" which is an accessible name that the screen reader should refer to this with
+  * Has `aria-valuenow` and `aria-valuetext` properties that update to indicate the current volume level (so the screen reader can read it)
+  * Has `aria-live` set to `polite`, which means that the screen reader should wait until the user is idle to send `aria-valuenow` updates to the user
+
+### The problem and the solution
+
+Now let's talk about how screen reader accessibility broke in Video.js 5. First `VolumeMenuButton` replaced `MuteToggle` and `VolumeControl` on the `ControlBar`. `VolumeMenuButton` was set to mimic `MuteToggle` when clicked. It would also show the `VolumeControl` on mouseover or focus. This was a problem because a `VolumeControl` was a now a child of a button, and buttons should not contain other controls. So to the screen reader and to the DOM there are two `MuteToggle button` controls. When visually there is a `VolumeControl` and a `MuteToggle`. Below you can see a gif of this behavior in action:
+
+![Before The Fix](before-the-fix.gif)
+
+The solution to this problem was to use a regular `div` to house the `MuteToggle` and `VolumeControl`. This regular `div` would have no role or control text so that it would be invisible to a screen reader. From that point forward we just needed to mimic the old UI. For those who are wondering, this new `Component` is called the `VolumePanel`. See the new screen reader behavior in a gif below:
+
+![After The Fix](after-the-fix.gif)
+
+### Outlines
+
+On top of fixing an accessibility issue with our controls, we decided to remove the following CSS from Video.js:
```

Comment:
"On top of fixing this accessibility issue with our controls, we also decided to remove the ...." (restoring outlines is another accessibility fix for the controls)